static/app/views/admin/options.tsx

```diff
@@ -174,6 +174,38 @@ const performanceOptionDefinitions: Field[] = [
     max: Number.MAX_SAFE_INTEGER,
     step: 1,
   },
+  {
+    key: 'performance.issues.consecutive_db.problem-creation',
+    label: t('Consecutive DB Problem Creation Rate'),
+    help: t(
+      'Controls the overall rate at which performance problems are detected by the Consecutive DB detector.'
+    ),
+    ...HIGH_THROUGHPUT_RATE_OPTION,
+  },
+  {
+    key: 'performance.issues.consecutive_db.la-rollout',
+    label: t('Limited Availability Detection Rate'),
```

Comment:
```suggestion
    label: t('Consecutive DB Limited Availability Detection Rate'),
```

We should prob include the issue type in these labels so we can differentiate (I noticed the name isn't included for the N+1 API Calls as well, so that's gonna make the UI a little confusing, we might as well update those labels too)

Reply:
Hmm, why not remove it, because its all under a heading anyways?
![image](https://user-images.githubusercontent.com/44422760/211388306-042f2662-7734-4bb8-acac-05cda4daaa17.png)